plugins/usbpro/OpenDeckDevice.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * OpenDeckDevice.cpp
+ * An OpenDeck Device
+ * Copyright (C) 2021 Simon Newton, Igor Petrovic
+ *
+ * This device creates a single output port
+ */
+
+#include <iomanip>
+#include <ios>
+#include <iostream>
+#include <string>
+
+#include "common/rpc/RpcController.h"
+#include "ola/Callback.h"
+#include "ola/Constants.h"
+#include "ola/Logging.h"
+#include "plugins/usbpro/OpenDeckDevice.h"
+#include "plugins/usbpro/UsbProWidgetDetector.h"
+
+namespace ola {
+namespace plugin {
+namespace usbpro {
+
+using ola::plugin::usbpro::Request;
+using ola::plugin::usbpro::Reply;
+using ola::rpc::RpcController;
+using std::string;
+
+/*
+ * Create a new device
+ * @param owner  the plugin that owns this device
+ * @param name  the device name
+ * @param dev_path  path to the pro widget
+ */
+OpenDeckDevice::OpenDeckDevice(ola::PluginAdaptor *plugin_adaptor,
+                                     ola::AbstractPlugin *owner,
+                                     const string &name,
+                                     OpenDeckWidget *widget,
+                                     OLA_UNUSED uint16_t esta_id,
+                                     OLA_UNUSED uint16_t device_id,
+                                     uint32_t serial,
+                                     uint16_t firmware_version,
+                                     unsigned int fps_limit):
```

Comment:
Can you align all these arguments like so:
```suggestion
OpenDeckDevice::OpenDeckDevice(ola::PluginAdaptor *plugin_adaptor,
                               ola::AbstractPlugin *owner,
                               const string &name,
                               OpenDeckWidget *widget,
                               OLA_UNUSED uint16_t esta_id,
                               OLA_UNUSED uint16_t device_id,
                               uint32_t serial,
                               uint16_t firmware_version,
                               unsigned int fps_limit):
```

Reply:
done